runtime/runtime.go

```diff
@@ -312,6 +312,20 @@ func Metadata(_, _ int32) int64 {
 		Metadata()
 }
 
+//go:export Metadata_at_version
+func MetadataAtVersion(dataPtr int32, dataLen int32) int64 {
+	return runtimeApi().
+		Module(metadata.ApiModuleName).(metadata.Module).
+		MetadataAtVersion(dataPtr, dataLen)
+}
+
+//go:export Metadata_versions
```

Comment:
Function name must be `Metadata_metadata_versions`
https://spec.polkadot.network/chap-runtime-api#sect-rte-metadata-metadata-versions

Reply:
Fixed